app.py

```diff
@@ -97,11 +97,13 @@ async def submit_form(form: SubmitIn,
                      redirect_url=redirect_url)
 
 
-@app.post('/webhook/interceptum-post/', response_model=SubmitOut)
-async def interceptum_post_form(form_dict: Dict,
-                                background_tasks: BackgroundTasks) -> SubmitOut:
+@app.post('/webhook/interceptum-post/')
+async def interceptum_post_form(background_tasks: BackgroundTasks,
+                                request: Request) -> SubmitOut:
     """Currently unusable."""
     # TODO: Map interceptum input to background task input
+    xml = (await request.form()).get('VALUES_XML')
+    form_dict = interceptum.xml_to_form_values(xml)
     if credentials.USE_WEBHOOK:
         background_tasks.add_task(background_processing, form_dict)
```

Comment:
We need to save the `form_dict` object to the database as well, similar to what we did in `/api/submit` since we moved that out of background_tasks